cdc/log.cc

```diff
@@ -389,12 +390,24 @@ bytes log_data_column_deleted_elements_name_bytes(const bytes& column_name) {
     return to_bytes(cdc_deleted_elements_column_prefix) + column_name;
 }
 
+// Makes sure that number of sstables won't grow too large
+static int calculate_window_size(int ttl) {
+    // What's the minimum window that won't create more than 24 sstables.
+    auto window_size = std::chrono::seconds(ttl / 24 + (ttl % 24 == 0 ? 0 : 1));
+    return std::chrono::ceil<std::chrono::minutes>(window_size).count();
```

Comment:
This is a really elaborate way to divide by 60...
Couldn't this entire function be just written as:
    return ttl / 60 / 24
Or maybe if you are really bothered by the exact division or truncation or something,
    return ttl / 60 / 24 + 1

If the function was this simple, inlining it below will make even more sense.

Reply:
I think using the right types actually has a value. It documents the code.